provider/oci/oci.go

```diff
@@ -236,13 +273,19 @@ func (p *OCIProvider) ApplyChanges(ctx context.Context, changes *plan.Changes) e
 		return nil
 	}
 
-	for zoneID, ops := range opsByZone {
-		if _, err := p.client.PatchZoneRecords(ctx, dns.PatchZoneRecordsRequest{
-			CompartmentId:           &p.cfg.CompartmentID,
-			ZoneNameOrId:            &zoneID,
-			PatchZoneRecordsDetails: dns.PatchZoneRecordsDetails{Items: ops},
-		}); err != nil {
-			return err
+	// Loop zones to get zone scope for zone patch records
```

Comment:
This comment doesn't seem to match what the code is doing. The code is applying a set of record changes to the zone.